### PR TITLE
fix(#2012): Adding minor padding to bottom of Scrollable

### PR DESCRIPTION
**Pull request by ArakTaiRoth**
# Before (the change)

image

# After (the change)

image

## Make sure that you've checked the boxes below before you submit the PR

- [x] I have read and followed the [setup steps](https://goa-dio.atlassian.net/wiki/spaces/DS/pages/3086385489/Contribution+-+web+content#%5BinlineExtension%5DCode-contributions)
- [x] I have created necessary unit tests
- [x] I have tested the functionality in both React and Angular.

## Steps needed to test

Create a modal using the actions slot, and include a form item in the content.

### CODE REVIEW

File: libs/web-components/src/components/scrollable/Scrollable.svelte [hunk 1]
```diff
@@ -44,7 +44,7 @@
   overflow-y: ${direction === "vertical" ? "auto" : "hidden"};
   overflow-x: ${direction === "horizontal" ? "auto" : "hidden"};
   margin: ${vpadding} 0;
-  padding: 0 ${hpadding};
+  padding: 0 ${hpadding} 2px;
```
Comment (chrisolsen) [inline libs/web-components/src/components/scrollable/Scrollable.svelte:47]: This should be 3px

**Review by syedszeeshan — APPROVED** [review]

**Pull request merged**